### PR TITLE
remove NOW runtime constant

### PR DESCRIPTION
**Pull request by edquist**

### CODE REVIEW

File: koji-blame [hunk 1]
```diff
@@ -101,7 +100,7 @@ def parse_cli(args):
     parser = argparse.ArgumentParser()
     parser.add_argument('-s', '--since', action=DateAction, dest='since', default=EPOCH,
                         help="List packages since YYYY-MM-DD")
-    parser.add_argument('-u', '--until', action=DateAction, dest='until', default=NOW,
+    parser.add_argument('-u', '--until', action=DateAction, dest='until', default=None,
```
Comment (brianhlin) [inline koji-blame]: Isn't the default default `None`?

Comment (edquist) [reply koji-blame]: Looks like yes:
https://docs.python.org/2/library/argparse.html#default

So yeah i'm on board with dropping the `default` parameter.

Comment (brianhlin) [reply koji-blame]:
```suggestion
    parser.add_argument('-u', '--until', action=DateAction, dest='until',
```

**Review by brianhlin — APPROVED** [review]
LGTM

**Pull request merged**